samples/excel/85-preview-apis/chart-data-source.yaml

```
@@ -0,0 +1,129 @@
+order: 6
+id: excel-chart-data-source
+name: Chart series data source
+description: This sample shows how to get information about the data source of a chart series.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#log-chart-series-source").click(() => tryCatch(logChartSeriesSource));
+
+        async function logChartSeriesSource() {
+          await Excel.run(async (context) => {
+            // This method retrieves the data source string and type values from
+            // the chart on the Sample worksheet.
```

Suggestion to clarify between the chart and chart series.
```suggestion
            // This method retrieves the data source information of a chart series in the Sample worksheet.
```